### PR TITLE
fix: upgraded versions.tf to include minor bumps from tpg v5

### PR DESCRIPTION
**Pull request by g-awmalik**

### CODE REVIEW

File: modules/cos-coredns/versions.tf [hunk 1]
```diff
@@ -20,11 +20,11 @@ terraform {
 
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.53, < 5.0"
+      version = ">= 3.53, < 6"
```
Comment (smaftoul) [inline modules/cos-coredns/versions.tf:23]: Why would we block to version 6 ?
We don't know what's included in version 6 yet, and the module may still work.
Currently, it's "< 5.0" , and it's blocking terraform setup to update to provider v5, it's the only terraform google module that I use that does this.

**Comment by bharathkkb** [comment]
@g-awmalik are the test failures related or can we merge this?

**Comment by apeabody** [comment]
> @g-awmalik are the test failures related or can we merge this?

I just merged https://github.com/terraform-google-modules/terraform-google-container-vm/pull/123 cleanly, let see if we now get a green run. If not, likely an incompatibility with TPG 5 which needs to be investigated.

**Review by apeabody — APPROVED** [review]

**Comment by apeabody** [comment]
@bharathkkb @g-awmalik all green now :)

**Pull request merged**